ndbench-dyno-plugins/src/main/java/com/netflix/ndbench/plugin/dyno/DynoJedisGetSetPipeline.java

```
@@ -111,4 +111,32 @@ public String writeSingle(String key) throws Exception {
         return jedisUtils.pipelineWrite(key, dataGenerator, MAX_PIPE_KEYS, MIN_PIPE_KEYS);
     }
 
+    /**
+     * Perform a bulk read operation
+     * @return a list of response codes
+     * @throws Exception
+     */
+    public List<String> readBulk(final List<String> keys) throws Exception {
```

Why do we care to implement bulk operations in the current plugins?

---

just retrofitting to existing plugins. Alternatively we can throw a UnsupportedOperationException.

---

Lets throw `UnsupportedOperationException` so that plugin owners can implement based on what they are looking for

---

#resolved